conftest.py

```diff
@@ -16,23 +16,40 @@ def pytest_addoption(parser):
         default=False,
         help="Run HDFS tests.",
     )
+    parser.addoption(
+        "--runwip",
+        action="store_true",
+        default=False,
+        help="Run WIP tests."
+    )
```

Comment:
Can we add some useful comments to say that this code skips WIP tests, and also that the way to mark a test as WIP is pytestmark = pytest.mark.runwip (took me a while to work that out)